src/components/document-metadata-editor.js

```diff
@@ -86,15 +83,13 @@ function DocumentMetadataEditor({ documentRevision, onChanged }) {
   const mergedTags = new Set([...defaultTags, ...documentRevision.tags, ...tagSuggestions]);
 
   const renderComponent = () => {
-    const docLanguage = languageNameProvider.getData(language)[documentRevision.language];
-
     switch (mode) {
       case MODE_PREVIEW:
         return (
           <div>
             <span>{t('common:title')}:</span> <span>{documentRevision.title}</span>
             <br />
-            <span>{t('common:language')}:</span> <span><CountryFlagAndName code={docLanguage.flag} name={docLanguage.name} /></span>
+            <span>{t('common:language')}:</span> <span><LanguageFlagAndName language={language} /></span>
```

Comment:
This should be document language instead of UI language.

Reply:
Oh, yes. Thanks, fixed.